specifications/Privateform-FRD.md

```diff
@@ -0,0 +1,779 @@
+# Privateform - Functional Requirements Document
+
+This document describes functional requirements for the [PrivateForm](https://form.blindnet.io/).
+
+## Background
+
+PrivateForm is a web application for doctors. It allows doctors to securely collect data from their patients in an end-to-end encrypted workflow. Patients access online forms where they answer a set of questions, after which the form is encrypted and sent to PrivateForm. There are two possible forms for patients to fill (COVID-19 and general medical form). 
+
+Doctors have accounts on PrivateForm Light, where they can login, see and update all their patient’s data. For more details see interaction and design documents.
+
+## Glossary
+
+_Server_ - PrivateForm Light backend server.
+
+_Doctor_ - Registered user of the Server.
```

Comment:
In the devkit specs we will have different vocabulary. Here, see if we can quickly rename concepts so that `doctor` becomes `consumer`, `patient` becomes `submitter`, and `medical data` becomes just `data`, so that we have generic specs in the same way in which privateform has evolved into a generic product. Not urgent I think.

Reply:
Done